configs/unet/unet-s5-d16_fcn_4xb4-160k_hsidrive-192x384.py

```diff
@@ -0,0 +1,53 @@
+_base_ = [
+    '../_base_/models/fcn_unet_s5-d16.py', '../_base_/datasets/hsi_drive.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
+]
+crop_size = (192, 384)
+data_preprocessor = dict(
+    type='SegDataPreProcessor',
+    size=crop_size,
+    mean=None,
+    std=None,
+    bgr_to_rgb=None,
+    pad_val=0,
+    seg_pad_val=255)
+
+model = dict(
+    data_preprocessor=data_preprocessor,
+    backbone=dict(in_channels=25),
```

Comment:
Since it's a hyperspectral image dataset, it's better to explain the modification to the `in_channel`. You might add a brief introduction to the docstring of dataset class (mmseg/datasets/hsi_drive.py) or dataset preparation document (docs/zh_cn/user_guides/2_dataset_prepare.md).

Reply:
Added brief explanation in mmseg/datasets/hsi_drive.py